### DISCUSSION TITLE
sensor2img - support for sonar sensor

### DISCUSSION THREAD

**Discussion by Mateus224** [dfki-ric/pytransform3d#127] [Feature discussions]
enhancement:
A more general form for the camera projection. 
implementing sensor2image for sonar sensor (single- and multibeam) too.

**Comment by AlexanderFabisch** [comment]
Hi, that is not my field of expertise and the camera module is not used very often at the moment. So I won't work on it. However, if you want to contribute something here, I'd review pull requests.
